ims/exception/file_system_exceptions.py

```diff
@@ -0,0 +1,129 @@
+from abc import ABCMeta
+
+from exception import FileSystemException
+
+
+# This exception should be raised when Image is not found in file system
+class ImageNotFoundException(FileSystemException):
+    @property
+    def status_code(self):
+        return 404
+
+    def __init__(self, name):
+        self.name = name
```

Comment:
Very minor style/readability note, but it seems what you actually pass to this is actually an image ID, not an image name.